packages/hd/package.json

```diff
@@ -23,7 +23,7 @@
     "@ckb-lumos/bi": "0.24.0-next.0",
     "@ckb-lumos/codec": "0.24.0-next.0",
     "@ckb-lumos/crypto": "0.24.0-next.0",
-    "bn.js": "^5.1.3",
+    "bn.js": "^4.12.0",
```

Comment:
Why is `bn.js` downgraded?

Reply:
The `elliptic` library in use depends on the `bn.js@4.12.0`. To prevent multiple versions of `bn.js`, this PR downgrades `bn.js` from 5.1 to 4.12

Reply:
Got it